static/openapi/hss-spec.json

```diff
@@ -0,0 +1,3701 @@
+        "/api/hss/v1/version": {
+            "get": {
+                "tags": [
+                    "Version"
+                ],
+                "summary": "Get version",
+                "description": "Get the current version.",
+                "operationId": "get_version_api_hss_v1_version_get",
+                "responses": {
+                    "200": {
+                        "description": "Successfully returned the version.",
+                        "content": {
+                            "application/json": {
+                                "schema": {},
+                                "example": {
+                                    "version": "1.0"
+                                }
+                            }
+                        }
+                    },
+                    "401": {
+                        "description": "Error: Unauthorized.",
+                        "content": {
+                            "application/json": {
+                                "example": {
+                                    "detail": "Unauthorized"
+                                }
+                            }
+                        }
+                    }
+                },
+                "security": [
+                    {
+                        "APIKeyHeader": []
+                    }
+                ]
+            }
+        },
+        "/api/hss/v1/health": {
+            "get": {
+                "tags": [
+                    "Health"
+                ],
+                "summary": "Get application health",
+                "description": "Get the current application health status.",
+                "operationId": "get_health_api_hss_v1_health_get",
+                "responses": {
+                    "200": {
+                        "description": "Successfully returned the health.",
+                        "content": {
+                            "application/json": {
+                                "schema": {},
+                                "example": {}
+                            }
+                        }
+                    }
+                }
+            }
+        },
+        "/api/hss/v1/models": {
+            "get": {
+                "tags": [
+                    "Models"
+                ],
+                "summary": "Get Models",
+                "operationId": "get_models_api_hss_v1_models_get",
+                "responses": {
+                    "200": {
+                        "description": "Successful Response",
+                        "content": {
+                            "application/json": {
+                                "schema": {
+                                    "items": {
+                                        "$ref": "#/components/schemas/Model"
+                                    },
+                                    "type": "array",
+                                    "title": "Response Get Models Api Hss V1 Models Get"
```

Comment:
All titles should be sentence case, not title case. Also, we aren't using "service" in names externally, so don't use "HSS."